.editorconfig

```diff
@@ -0,0 +1,372 @@
+root = true
+
+# All files
+[*]
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+
+# Xml files
+[*.xml]
+indent_size = 2
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# Indentation and spacing
+indent_size = 4
+tab_width = 4
+
+# New line preferences
+end_of_line = crlf

```

Comment:
why not `lf`? Are you changing all files to `lf` or `crlf`?

Reply:
I have no preferences. Making it `lf`